e2e/setup/serializers/image/diff-builder.ts

```diff
@@ -0,0 +1,42 @@
+import sharp from 'sharp';
+import {SharpService} from './sharp';
+
+import type {SharpContext} from './sharp';
+
+type DiffBuilderConfig = {
+  diffPath: string;
+  img1: SharpContext;
+  img2: SharpContext;
+  diffBuffer: Buffer;
+};
+
+export class DiffImageComposer {
```

Comment:
Smell (for future rework): the OOP approach (to be persize: creation of object composer that can not be reused but only saved sounds redundant) does not structure or give benefits to the purpose of the module.